### PR TITLE
docs: Add screenshots for troubleshooting device trust in web UI

### LINKED ISSUE #45377: Document device trust web green/yellow shield
Mention the green/yellow shield in the public device trust docs.

## Applies To

Applies to v16+ device trust pages.

## Details

The green shield means the web session is using a trusted device.

The yellow shield means the web session is not using a trusted device, but Teleport detected that this may impact the user's accesses (ie, either cluster or roles require device trust).

## How will we know this is resolved?

A brief explanation is good enough for now.

## Related Issues

- https://github.com/gravitational/teleport/issues/43126

### PR DESCRIPTION
**Pull request by webvictim**
Adds some information about the green and yellow shield icons in the web UI, indicating the status of the device trust session.

Fixes #45377

### CODE REVIEW

**Review by stevenGravy — APPROVED** [review]
As the prose recommends would just make it Device Trust

**Comment by zmb3** [comment]
Personally, I'm not a fan of adding screenshots to the docs.

In some cases (especially large/complex workflows) it is unavoidable, but in the general case we should strive to make the UI as self-explanatory as possible.

Screenshots go out of date as the UI changes and tend to bloat the size of the repo. In this case, I think we can probably get by with a simple plain-text description of the device trust indicator.

File: docs/img/device-trust/device-trust-web-ui-failure-1.png
[the remark this answers is not in the document]
Comment (webvictim) [reply docs/img/device-trust/device-trust-web-ui-failure-1.png:1]: Haha, whoops!

**Comment by webvictim** [comment]
@zmb3 Would you consent to just having the icons rather than full screenshots? I'm thinking that for some folks where English isn't their native language, the explanation of symbol/colour might be more accurately conveyed by a picture than a description.

**Review by codingllama — COMMENTED** [review]
Many thanks, Gus!

**Review by codingllama — APPROVED** [review]

**Review by mmcallister — APPROVED** [review]

**Pull request merged**